### DISCUSSION TITLE
Blogpost #1

### DISCUSSION THREAD

**Discussion by lucariebel** [lucariebel/online-shop#2] [General]
## Produktvision
Unsere Produktvision ist es, eine einzigartige Online-Plattform zu schaffen, die sowohl Sofortkauf- als auch Auktionsfunktionen bietet. Wir bieten eine breite Palette an coolen Produkten an, von Streetwear bis hin zu getunten Autos. Unser Ziel ist es, eine Anlaufstelle für Trendsetter zu sein.
## Unser Team
- Stefan Schäfer
- Sebastian Albert
- Philip Sagawe
- Luca Riebel
- Oskar Bartsch

Zum aktuellen Zeitpunkt wurden noch keine spezifischen Rollen für die Entwickler festgelegt. Diese werden zu einem späteren Zeitpunkt nachgereicht.
## Tech-Stack
- Frontend: Angular
- Backend: ASP.NET
- Datenbank: SQLite
- Projektmanagement: Jira
## Scrum Setup
Für die Projektplanung wird die Plattform Jira genutzt. Die Sprintdauer beträgt zwei Wochen, wobei der Wechsel der Sprints alle zwei Wochen dienstags erfolgt. Die Dailies finden wöchentlich jeweils montags und donnerstags statt.
## Sprint Backlog #1
Für den ersten Sprint wurden folgende Aufgaben geplant:
- Angular Grundlagen lernen
- Projekt erstellen
- Packages hinzufügen
- Header einfügen
- Anmelde-Dialog erstellen
## Kanbanboard
Da erst morgen, am 15.10.2024, der Sprintwechsel ist, wird das Kanbanboard nachgereicht.

**Comment by ChristophNie** [comment]
Hallo, 
Habt ihr dann auch vor, dass andere User diverse Artikel hochladen und anbieten können, oder wird das Angebot nur von euch erstellt? Falls auch andere Nutzer Artikel hochladen können, soll es dann auch eine Filter Möglichkeit geben wie Standort o.ä?
LG 
Team Platzverwaltung

**Reply by lucariebel** [reply]
Hi,
ja, jeder User wird die Möglichkeit haben eigene Artikel anbieten zu können. Entweder zum Sofortkauf oder zur Autkion.
Filter zu einer präziseren Suche nach Artikeln sind auch geplant.
LG Luca
